suites/squid/common/regression/rados/rados_4-node-ecpools.yaml

```diff
@@ -0,0 +1,738 @@
+# Suite is to be used to verify the EC 2+2 and EC 8+6 with MSR config on 4 nodes
+# conf: conf/squid/rados/5node-1client-ecpool-replica1.yaml
+# Deployment: suites/squid/common/regression/ecpool-replica1-deploy-and-configure.yaml
+tests:
+
+  - test:
+      name: Configure email alerts
+      module: rados_prep.py
+      polarion-id: CEPH-83574472
+      config:
+        email_alerts:
+          smtp_host: smtp.corp.redhat.com
+          smtp_sender: ceph-iad2-c01-lab.mgr@redhat.com
+          smtp_port: 25
+          interval: 10
+          smtp_destination:
+            - pdhiran@redhat.com
+          smtp_from_name: EC 8+6@4 MSR rules cluster alerts
+      desc: Configure email alerts on ceph cluster
+
+  - test:
+      name: Enable logging to file
+      module: rados_prep.py
+      config:
+        log_to_file: true
+      desc: Change config options to enable logging to file
+
+  - test:
+      name: Set configs for 4 node cluster
+      desc: Set configs for 4 node cluster
+      module: test_cephadm.py
+      config:
+        steps:
+          - config:
+              command: shell
+              args:
+                - ceph
+                - config
+                - set
+                - mon
+                - mon_osd_down_out_subtree_limit
+                - host
+
+  - test:
+      name: EC pool 2+2@4 LC
+      module: test_four_node_ecpool.py
+      polarion-id: CEPH-83575858
+      config:
+        ec_pool:
+          profile_name: ec22_profile
+          pool_name: test_ec_pool
+          k: 2
+          m: 2
+          force: true
+          plugin: jerasure
+          crush-failure-domain: host
+        remove_host: true
+        delete_pools:
+          - test_ec_pool
+      desc: 2+2@4 EC pool life cycle with serviceability scenarios
+
+  - test:
+      name: EC 2+2 Pool Recovery Improvement
+      module: pool_tests.py
+      polarion-id: CEPH-83573852
+      config:
+        ec_pool_recovery_improvement:
+          create: true
+          pool_name: ec_pool_recovery
+          k: 2
+          m: 2
+          pg_num: 32
+          plugin: jerasure
+          crush-failure-domain: host
```

Comment:
Update the duration with max_objects. 

This will reduce the run time of test case. esp recovery time

Reply:
done